src/main/java/io/cryostat/net/web/http/api/v1/RecordingGetHandler.java

```diff
@@ -87,7 +87,7 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
         String recordingName = ctx.pathParam("recordingName");
         try {
             Path archivedRecording = recordingArchiveHelper.getRecordingPath(recordingName).get();
-            ctx.response().sendFile(archivedRecording.normalize().toAbsolutePath().toString());
+            ctx.response().sendFile(archivedRecording.toString());
```

Comment:
And here - what if the returned path is null?

Reply:
I'll resolve the null case inside the `getRecordingPath` function.